src/Endatix.Infrastructure/Integrations/Slack/SlackClient.cs

```diff
@@ -19,28 +21,64 @@ public class SlackClient : INotificationHandler<SubmissionCompletedEvent>, ISlac
     private readonly ILogger<SlackClient> _logger;
     private readonly IRepository<Form> _formRepository;
```

Comment:
It seems we don't need the repo here and in the constructor as we're not injecting it during the object creation.

Reply:
Done